DOC-2080: TinyMCE 6.5.1 community docs release (#2791)
tinymce/tinymce-docs: modules/ROOT/pages/changelog.adoc

NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].

== 6.5.1 - 2023-06-19

=== Fixed
* Fixed a regression where pasting an image url would result in the url being inserted as plain text instead of the image being inserted.

> * It was not possible to press space to insert a space character inside a summary element on Firefox.

Reviewer: ```suggestion
* Pressing "spacebar" to insert a space character within a summary element for an Accordion component was not possible on Firefox.
```

Reviewer: @brian-forte just a suggestion more than anything.
